typed_python/compiler/tests/alternative_compilation_test.py

```diff
@@ -1075,7 +1217,7 @@ def f_delattr2(x: A):
             self.assertEqual(f_getattr2(v), "4")
             self.assertEqual(f_getattr2(v), c_getattr2(v))
             f_delattr1(v)
-            # exception types are different
+            # TODO: exception types are different.  Should this be made consistent?
```

Comment:
yes. we used to not be able to raise anything but TypeError, so our old tests did the wrong thing. But now we can throw real exceptions, so we should be trying to get this right.

Reply:
This has been done.